app/components/geoblacklight/attribute_table_component.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Geoblacklight
+  ##
+  # A component for rendering attribute table
+  #
+  class AttributeTableComponent < ViewComponent::Base
+    attr_reader :show_attribute_table
```

Comment:
Why do you have this if you aren't using it?

Reply:
You are correct : ) will remove

Reply:
Have removed